src/libs/antares/study/include/antares/study/parts/hydro/container.h

```diff
@@ -28,6 +28,60 @@
 
 namespace Antares::Data
 {
+
+    //! The maximum number of days in a year
+constexpr size_t dayYearCount = 366;
+
+struct HydroDailyLoadDemand
+{
+    //! Net demand, for each day of the year, for each area
+    double DLN = 0.;
+    //! Daily local effective load
+    double DLE = 0.;
+};
+
+struct HydroMonthlyDemandGeneration
+{
+    //! Monthly local effective demand
+    double MLE = 0.;
+    //! Monthly optimal generation
+    double MOG = 0.;
+    //! Monthly optimal level
+    double MOL = 0.;
+    //! Monthly target generations
+    double MTG = 0.;
+};
+
+//!  Hydro Management Data for a given area
+struct TimeDependantHydroManagementData
+{
+    std::array<HydroDailyLoadDemand, dayYearCount> daily{0};
+    std::array<HydroMonthlyDemandGeneration, 12> monthly{0};
+};
+
+//! Area Hydro Management Data for a given year
+struct AreaDependantHydroManagementData
```

Comment:
**struct TimeDependantHydroManagementData** : 
This **struct** is used only during hydro ventilation (hydro management if you prefer).
It's not used when checking hydro data.
So we could ask : why is it defined here, and not in class **HydroManagement** ?

Same for other struct above.